cmd/list.go

```diff
@@ -27,13 +31,15 @@ func init() {
 
 func List(cmd *cobra.Command, args []string) {
```

Comment:
As is now, panics if just language is given (`uru list [lang]`)... should either verify that both lang and module type are given or have `uru list [lang]` print all modules (evasion, encoders, injectors)

Reply:
So, i have been working on the list command.

I have changed the function so now it looks for in templates in binary using this (https://github.com/guervild/uru/pull/7/files#r1072138995).


I also modified the code to check for args, so there is no more crash. Possible commands are like this:
```
uru encoders go
uru options
```

The only problem i have is the way you changed all the names (in the injectorFactory). I prefered the old way ^^

I means, in the name it is important for me to have a "local" directory, as for the template, if one day we create "remote" injections (i have been working on this, not pushed).
But, all my names were not "clean", i have to chose if we use camelcase, "-", or "_" in the artifact name (suggestions taken :p)

So, next time i will think about name, rework all the name, and push the changes

Reply:
also i might change "injector" to "injectors"

Reply:
I changed to something that look in the binary since i did not add a feature that look for templates in a chosen directory (could be a cool feature to add)

Reply:
you were more purposeful with your naming, so adopting your scheme is a good idea.